$NetBSD: patch-src_openrct2_CMakeLists.txt,v 1.8 2023/09/02 19:59:49 triaxx Exp $

GCC 9 does not require linking with -lstdc++fs to use types and functions in
<filesystem> now (https://gcc.gnu.org/gcc-9/changes.html).

--- src/openrct2/CMakeLists.txt.orig	2022-10-05 10:07:57.000000000 +0000
+++ src/openrct2/CMakeLists.txt
@@ -37,13 +37,6 @@ endif ()
 set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "")
 SET_CHECK_CXX_FLAGS(${PROJECT_NAME})
 
-# GCC / Clang likes us to pass the -lstdc++fs flag to link C++17 filesystem implementation.
-if (NOT MINGW AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD" AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD")
-    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-        target_link_libraries(${PROJECT_NAME} stdc++fs)
-    endif()
-endif()
-
 if (NOT DISABLE_NETWORK OR NOT DISABLE_HTTP)
     if (WIN32)
         target_link_libraries(${PROJECT_NAME} bcrypt)
